### DISCUSSION TITLE
Daily 12/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#259] [Daily Sprint]
- Qué hice ayer?
- Qué haré hoy
- Dificultades o Impedimentos

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve revisando los formularios de login y registración para para que sean reactivos. También  trate de agregarle validación pero no me tiraba error y preferí dejarlo para el finde o la semana que viene.
Hoy voy a tratar de hacer el servicio para Login, aunque se me presentan muchas duda. Voy a ver el video de la explicación que hicimos en la reunión y a leer los apuntes nuevamente, sino pediré ayuda.

**Comment by rpgrca** [comment]
Estos días estuve bastante ocupado en el trabajo pero hoy alcancé a agregar el modelo de Selección para el carrito ya que no tiene que ir el producto directamente y reacomodar todo para usar la selección más algunos arreglos simples de formato.

**Comment by BraianTroncoso** [comment]
Me tuve que conectar a una charla sobre Game Design de un curso optativo que estoy tomando a demás ya había finalizado la parte de servicios.
